Assets/Scripts/Parallax_Script.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BG_Parallax : MonoBehaviour {
+
+	private float length, startposX;
+	//startposY;
+	public GameObject cam;
+	public float parallaxEffect;
```

Comment:
What does parallaxEffect mean in terms of values? I understand what it is conceptually, but I'm confused as to what this would mean if like parallaxEffect = 0 vs. 1000.